drizzlepac/processInput.py

```diff
@@ -131,6 +131,23 @@ def setCommonInput(configObj, createOutwcs=True, overwrite_dict={}):
         # Update configObj with values from mpars
         cfgpars.mergeConfigObj(configObj, mdriztab_dict)
     
+    # add warnings for kernels that may not work as intended
+    kernels_w_warnings = ["gaussian", "lanczos3", "tophat"]
+    for kernel_param in overwrite_dict:
+        if overwrite_dict[kernel_param] in kernels_w_warnings:
+            if overwrite_dict[kernel_param] == "tophat":
+                log.warning(
+                    f'WARNING: It was found that kernel "{overwrite_dict[kernel_param]}" does not work as intended. '
+                    + f'It has been removed as a option. Using kernel "square" instead for {kernel_param}.'
```

Comment:
Do you want kernel 'square' to be promoted here, or should it be kernel 'turbo'?  It seems that 'turbo' is now the default.

Reply:
I happy to use turbo. I don't think that this will be a common scenario, and if it does, a user may see that their parameter is being ignored and set the kernel to a viable option in a follow-up run of the code.

Reply:
The default for the final drizzle should be 'square' and for CR-rejection it was 'turbo' for performance reasons.